src/server/test/user-auth.js

```diff
@@ -527,8 +527,8 @@ if (mongoose.connection.name === 'lookahead-test') {
               }
             )
             .end((err, res) => {
-              expect(res).to.have.status(400)
-              expect(res.body.message).to.equal('Incorrect Password')
+              expect(res).to.have.status(401)
+              expect(res.body.message).to.equal("Invalid Email or Password.")
```

Comment:
This is an email update - the message shouldn't be generic and status should be 400.